examples/msg.rs

```diff
@@ -1,4 +1,19 @@
 fn main() {
+    let res = "";
+    #[cfg(any(
+        target_os = "windows",
+        target_os = "macos",
+        all(
+            any(
+                target_os = "linux",
+                target_os = "freebsd",
+                target_os = "dragonfly",
+                target_os = "netbsd",
+                target_os = "openbsd"
+            ),
+            feature = "gtk3"
+        )
+    ))]
```

Comment:
Could we maybe do any unix platform and just add "not macos" check to that?
Using `target_family` unix check

Reply:
No, `unix` can be macOS or Android.

Reply:
`unix`, [for now](https://github.com/rust-lang/rust/issues/58590), can also be Fuscia.